code/utils/linear_modeling.py

```diff
@@ -18,16 +18,14 @@ def get_design_matrix():
     """
     data = nib.load('test_data.nii')
     n_trs = data.shape[-1]
-    X = np.ones((n_trs, 4))
+    X = np.ones((n_trs, 3))
     ss = ssm.SceneSlicer('test_data.nii','scenes.csv') 
     day_night, int_ext = ss.get_scene_slices()
-    X[:, 0] = day_night
-    X[:, 1] = int_ext
-    # X[:, 2] = pos
+    X[:, 1] = day_night
     X[:, 2] = np.linspace(-1, 1, n_trs)
```

Comment:
This isn't super important, but I would prefer to have the linear drift column next to the column of ones, just so all the columns that we are less interested in (i.e. intercept and linear drift) are next to each other